src/config/from_document.rs

```diff
@@ -75,15 +78,57 @@ fn process_schema_directives<T: DirectiveCodec<T> + Default>(
     res
 }
 
+fn process_schema_multiple_directives<T: DirectiveCodec<T> + Default>(
```

Comment:
We can avoid duplication right with `process_schema_directives`. I think having one of the two functions is better.

Reply:
essentially, these two methods are doing different things. 

`process_schema_directives` handles one specific directive and uses the last occurrence of it. `process_schema_multiple_directives`, by the other side, returns all directives with this occurrence.
I can use only one of'em and add a new boolean value to check if it should behave as `process_schema_multiple_directives` or `process_schema_directives`, but IMO this should be avoided.

plase let me know if you have something else in mind